scripts/run-draft-pr-scripts.ts

```diff
@@ -142,37 +142,12 @@ class DraftPRScriptRunner {
     
     this.prNumber = await this.findPRByBranch();
```

Comment:
- delete `setupPREnvironment`
- replace its usage with `findPRByBranch`
- move the logs from `setupPREnvironment` into `findPRByBranch`